### PR TITLE
This week's CFPs!

### PR DESCRIPTION
**Pull request by bennyvasquez**

### CODE REVIEW

File: draft/2024-05-15-this-week-in-rust.md [hunk 1]
```diff
@@ -88,8 +88,9 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 Are you a new or experienced speaker looking for a place to share something cool? This section highlights events that are being planned and are accepting submissions to join their event as a speaker.
 
-<!-- CFPs go here, use this format: * [**event name**](link to CFP)| Date CFP closes in YYYY-MM-DD | city,state,country | Date of event in YYYY-MM-DD -->
-<!-- or if none - *No Calls for papers or presentations were submitted this week.* -->
+* [EuroRust 2024](https://www.papercall.io/eurorust-2024)| Closes 2024-06-03 | Vienna, Austria & online | Event date: 2024-10-10
+* [Scientific Computing in Rust 2024](https://scientificcomputing.rs/)| Closes 2024-06-14 | online | Event date: 2024-07-17 - 2024-07-19
+* [Conf42 Rustlang 2024](https://www.papercall.io/conf42-rustlang-2024) | Closes 2024-07-22 | online | Event date: 2024-08-22
```
Comment (mariannegoldin) [inline draft/2024-05-15-this-week-in-rust.md:93]: Would it be possible to add this late-coming addition for the Rust Argentina community?
```suggestion
* [Conf42 Rustlang 2024](https://www.papercall.io/conf42-rustlang-2024) | Closes 2024-07-22 | online | Event date: 2024-08-22
* [Rust Argentina June 2024](https://sessionize.com/rust-argentina-june/) | Closes 2024-05-31 | Buenos Aires, AR | Event date: 2024-06-04
```

Comment (bennyvasquez) [reply draft/2024-05-15-this-week-in-rust.md:93]: Yes yes!

File: draft/2024-05-15-this-week-in-rust.md [hunk 2]
```diff
@@ -88,8 +88,10 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 Are you a new or experienced speaker looking for a place to share something cool? This section highlights events that are being planned and are accepting submissions to join their event as a speaker.
 
-<!-- CFPs go here, use this format: * [**event name**](link to CFP)| Date CFP closes in YYYY-MM-DD | city,state,country | Date of event in YYYY-MM-DD -->
-<!-- or if none - *No Calls for papers or presentations were submitted this week.* -->
+* [EuroRust 2024](https://www.papercall.io/eurorust-2024)| Closes 2024-06-03 | Vienna, Austria & online | Event date: 2024-10-10
+* [Scientific Computing in Rust 2024](https://scientificcomputing.rs/)| Closes 2024-06-14 | online | Event date: 2024-07-17 - 2024-07-19
```
Comment (mariannegoldin) [inline draft/2024-05-15-this-week-in-rust.md:92]: formatting tweak:
```suggestion
* [EuroRust 2024](https://www.papercall.io/eurorust-2024) | Closes 2024-06-03 | Vienna, Austria & online | Event date: 2024-10-10
* [Scientific Computing in Rust 2024](https://scientificcomputing.rs/) | Closes 2024-06-14 | online | Event date: 2024-07-17 - 2024-07-19
```

**Review by mariannegoldin — APPROVED** [review]
TY! Approved with one small formatting change.

**Pull request merged**